core/templates/pages/subtopic-viewer-page/subtopic-viewer-page.module.ts

```diff
@@ -28,6 +28,9 @@ import { OppiaAngularRootComponent } from
 import { SubtopicViewerNavbarBreadcrumbComponent } from
   // eslint-disable-next-line max-len
   'pages/subtopic-viewer-page/navbar-breadcrumb/subtopic-viewer-navbar-breadcrumb.component';
+import { SubtopicViewerNavbarPreLogoActionComponent } from
+  // eslint-disable-next-line max-len
```

Comment:
I think this is not required for import statements.

Reply:
done

Reply:
done